app/src/pages/playground/Playground.tsx

```diff
@@ -293,29 +298,39 @@ function PlaygroundContent() {
             </Suspense>
           ) : (
             <div css={playgroundInputOutputPanelContentCSS}>
-              <Accordion arrowPosition="start" size="L">
+              <DisclosureGroup expandedKeys={["input", "output"]}>
                 {templateLanguage !== TemplateLanguages.NONE ? (
-                  <AccordionItem title="Inputs" id="input">
-                    <View padding="size-200" height={"100%"}>
-                      <PlaygroundInput />
-                    </View>
-                  </AccordionItem>
+                  <Disclosure id="input">
+                    <DisclosureTrigger arrowPosition="start">
+                      Inputs
+                    </DisclosureTrigger>
+                    <DisclosurePanel>
+                      <View padding="size-200" height={"100%"}>
+                        <PlaygroundInput />
+                      </View>
+                    </DisclosurePanel>
+                  </Disclosure>
                 ) : null}
-                <AccordionItem title="Output" id="output">
-                  <View padding="size-200" height="100%">
-                    <Flex direction="row" gap="size-200">
-                      {instances.map((instance, i) => (
-                        <View key={i} flex="1 1 0px">
-                          <PlaygroundOutput
-                            key={i}
-                            playgroundInstanceId={instance.id}
-                          />
-                        </View>
-                      ))}
-                    </Flex>
-                  </View>
-                </AccordionItem>
-              </Accordion>
+                <Disclosure id="output">
```

Comment:
Missing a devider on the top on the second
<img width="875" alt="Screenshot 2025-01-02 at 7 47 07 PM" src="https://github.com/user-attachments/assets/7b6c6853-bfb6-4261-aa75-9f3aee4f9ccc" />

Reply:
Fixed